File: merge-sorted-array/naive/Driver.java
package naive;
public class Driver {

    public static void main(String[] args) {

        Solution sol = new Solution();
        int[] nums1 = {1,2,3,0,0,0};
        int m = 3;
        int[] nums2 = {2,5,6};
        int n = 3;

        int[] nums = sol.merge(nums1, m, nums2, n);
        for (int i = 0; i < nums.length; i++) {
          System.out.println(nums[i]);
        }
    }

}

File: merge-sorted-array/optimal/python/Driver.py
from Solution import Solution

def main():
    nums1 = [1, 2, 3, 0, 0, 0]
    m = 3
    nums2 = [2, 5, 6]
    n = 3

    sol = Solution()
    sol.merge(nums1, m, nums2, n)

    print(nums1)

if __name__ == "__main__":
  main()